src/model.py

```diff
@@ -483,6 +514,14 @@ def finalize(self):
         """
         self.logger.log_info("[vllm] Issuing finalize to vllm backend")
         self._shutdown_event.set()
+
+        # Signal shutdown to the response sender thread.
```

Comment:
Can we clearly define what is `_loop_thread` and `_response_thread`? These terms can mix up easily.

Reply:
How about `_generate_thread` and `_response_thread`?

Reply:
hmmm... generate sounds something specific to the vLLM framework.
We can probably do `_event_thread` and `_response_thread`?
Also, let's document the reason for including the response_thread in the comment in L118.

Reply:
updated